packages/amplify-app/src/index.js

```diff
@@ -346,19 +346,23 @@ async function createJSHelperFiles() {
 }
 
 async function createAndroidHelperFiles() {
-  const configJsonObj = { profile: 'default', envName: 'amplify', syncEnabled: true };
-  const configJsonStr = JSON.stringify(configJsonObj);
-  const configFile = path.join(process.cwd(), './amplify-gradle-config.json');
+  const configJsonObj = {
+    profile: 'default',
+    envName: 'amplify',
+    syncEnabled: true,
+  };
+  const configJsonStr = JSON.stringify(configJsonObj, null, 4);
+  const configFile = './amplify-gradle-config.json';
```

Comment:
Won't this fail for windows?

Reply:
Yes but I'm going to have all the fixes for windows support in a separate PR

Reply:
Changing this back. Absolute path is ok here since it is only for writeFileSync